### ISSUE TITLE
Speech act information

### ISSUE THREAD

**Issue by mitjanikolaus** [langcog/childes-db#54]
Would it be possible to add speech act information to the utterances table? It is for example present in the [New England corpus](https://childes.talkbank.org/access/Eng-NA/NewEngland.html), indicated by the %spa tag. Thanks in advance!

**Comment by smeylan** [comment]
Yes, we hope to add this at some point in the future. Get in touch if you want to help!

**Comment by mcfrank** [comment]
Good news — this is done in the 2026.1 release: the new import pipeline captures the %spa tier, so speech act codes (e.g. in the New England corpus) are now available on utterances. 2026.1 is published on the Redivis-hosted childes-db: https://stanford.redivis.com/datasets/b6q6-14tmdjgw8. Closing; please reopen or file a new issue if the %spa coding in 2026.1 doesn't cover what you need.
